onnxruntime/core/mlas/lib/power/qgemm_kernel_power10.cpp

```diff
@@ -521,31 +543,133 @@ MlasGemmQuantCopyPackA8x8(
     }
 }
 
-template<typename Vtype>
+template<typename Vtype, bool BIsSigned>
 void
 MlasGemmQuantCopyPackB8x8(
     MLAS_GEMM_QUANT_KERNEL_POWER10::PackedBType* D,
     const uint8_t* B,
     size_t ldb,
     size_t CountN,
     size_t CountK,
-    int32_t* ColumnSumBuffer,
-    bool BIsSigned
+    int32_t* ColumnSumBuffer
     )
 {
     const uint8_t BitFlipValue = (BIsSigned ? 0x80 : 0);
+    ORT_UNUSED_PARAMETER(BitFlipValue);
```

Comment:
why add `ORT_UNUSED_PARAMETER` here? isn't BitFlipValue used just below in the vmask assignment?

Reply:
Some compilers still emit an unused variable - even though it is not true.

Reply:
Suggest using `[[maybe_unused]]` attribute.